CHANGELOG.md

```diff
@@ -181,14 +181,14 @@ The call signature of `deduplicate` has changed. The previous call signature is
 - The `relation_alias` argument has been removed as the macro now supports `relation` as a string directly. If you were using `relation_alias` to point to a CTE previously then you can now pass the alias directly to `relation`.
 
 Before:
-```jinja
+```sql
```

Comment:
There were several places where I was not sure whether to use `sql` or `jinja` as the language identifier.
Which is better, `sql` or `jinja`?

For now, I'm trying to unify it to `sql`.